table/block_based/block_based_table_reader.cc

```diff
@@ -1715,6 +1715,9 @@ void BlockBasedTable::RetrieveMultipleBlocks(
       req_offset_for_block.emplace_back(0);
     }
     req_idx_for_block.emplace_back(read_reqs.size());
+
+    PERF_COUNTER_ADD(block_read_count, 1);
```

Comment:
Is there any advantage to using temporary/stack variables for this counters and then updating the PERF_COUNTER variables only once (rather than on every pass through the loop)?

Reply:
I don't think so. The PerfContext is thread local, so directly updating it shouldn't add much overhead.